grafana-plugin/src/models/loader/action-keys.ts

```diff
@@ -0,0 +1,5 @@
+export enum ActionKey {
+  IS_ADDING_NEW_COLUMN_TO_ALERT_GROUP = 'IS_ADDING_NEW_COLUMN_TO_ALERT_GROUP',
```

Comment:
In the near future we can probably group this by the page name or something simillar